docs/guide/getstarted/install.md

```diff
@@ -28,7 +28,12 @@ server and browser driver separately.
 
 The simplest way to get started is to use one of the NPM selenium standalone
 packages like: [vvo/selenium-standalone](https://github.com/vvo/selenium-standalone). After installing
-it (globally) you can run your server by executing:
+it (globally), install the default driver:
```

Comment:
This should be `install the default drivers:` since Chromedriver is installed as a part of the selenium-standalone npm package (in addition to firefox driver).

However, I'm not sure if this line, in general, is needed per the wdio documentation. I believe after installing selenium-standalone via npm users are instructed to first run `selenium-standalone install` before attempting to execute `selenium-standalone start`. 

@christian-bromann @junketjim I'm not convinced if this is good instruction/documentation for the wdio project or rather, hand holding for users installing the selenium-standalone package. Let me know what you think.